content/md/permissions-settings/access-rights-on-products.md

```diff
@@ -131,7 +131,7 @@ If a user **is owner of the product information** for the products that are in a
 ### The edition right
 If a user **has rights to only edit product information** for the products that are in a given category (ie his user groups have at least the `Allowed to edit products` permission), he will be able to:
 *   View the product information
-*   Create a product [draft](workflow.html) that will need to be approved by the product manager
+*   Create a product [draft](proposals-workflow.html#contribute-to-a-working-copy) that will need to be approved by the product manager
```

Comment:
I think this link will be better, what do you think?
```suggestion
*   Create a product [draft](proposals-workflow.html#statuses-of-products) that will need to be approved by the product manager
```

Reply:
As it's about the process to create a draft, I'm not sure that "statuses of products" is better than "contribute-to-a-working-copy".
But it's your documentation, so tell me what you prefer to put.

Reply:
You're totally right. My bad.